pkg/bbgo/environment.go

```diff
@@ -139,8 +139,36 @@ func (environ *Environment) Connect(ctx context.Context) error {
 
 			marketDataStore := store.NewMarketDataStore(symbol)
 			marketDataStore.BindStream(session.Stream)
-
 			session.marketDataStores[symbol] = marketDataStore
+
+			standardIndicatorSet := NewStandardIndicatorSet(symbol)
+			standardIndicatorSet.BindMarketDataStore(marketDataStore)
+		}
+
+		// move market data store dispatch to here, use one callback to dispatch the market data
+		// session.Stream.OnKLineClosed(func(kline types.KLine) { })
+
+		now := time.Now()
+		for symbol := range loadedSymbols {
+			marketDataStore, ok := session.marketDataStores[symbol]
+			if !ok {
+				return errors.Errorf("symbol is not defined")
+			}
+
+			for interval := range types.SupportedIntervals {
+				kLines, err := session.Exchange.QueryKLines(ctx, symbol, interval.String(), types.KLineQueryOptions{
+					EndTime: &now,
```

Comment:
Do we really need the pointer of `now`?

Reply:
yes, it's an optional parameter, we use nil to check if it's used.